IBX-431: Allowed to lock & unlock drafts in workflow (#109)
ezsystems/user-documentation: docs/publishing/editorial_workflow.md

    Contact your administrator or developer team to configure workflows.
    See [developer documentation](https://doc.ibexa.co/en/master/guide/workflow/) for information on how to do it.

## Review queue

You can view Content items which are in different stages under review on the 
**My dashboard** screen, in the Review queue table.

> The table will only show Content items that your Role has permissions to edit.

Reviewer: ```suggestion
The table only shows Content items that your Role has permissions to edit.
```

> If your installation is [configured to support draft locking](https://doc.ibexa.co/en/master/guide/workflow/#draft-locking), the table will 
> also inform you whether any reviewers are assigned and have claimed their drafts 
> for review.

Reviewer: ```suggestion
If your installation is [configured to support draft locking](https://doc.ibexa.co/en/master/guide/workflow/#draft-locking),
the table also informs you whether any reviewers are assigned and have claimed their drafts for review.
```

![Review queue in the dashboard](img/dashboard_review_queue.png)

In the Review queue table, you can access the draft for editing, review its history, 

> and unlock the draft or request that a lock is released.

Reviewer: ```suggestion
and unlock the draft or ask to release the lock.
```
...maybe. Subjunctive tends to be hard to read for non-fluent readers

## Draft editing

When you click the **Edit draft** icon and open a Content item that has been sent 
to you for review, you see the Events timeline that lists all the transitions 
that this content has gone through.

![Events timeline](img/workflow_events_timeline.png)

> If draft locking is supported, you will also see a message that confirms that the draft is now locked to you.

Reviewer: ```suggestion
If draft locking is supported, you also see a message that confirms that the draft is now locked to you.
```
